config/tags.js

```diff
@@ -6,6 +6,14 @@
 import { blue, green, orange, purple, red, yellow } from '../src/utils/colours'
 
 export const tags = {
+  oss: {
+    title: 'First Time Contributors',
+    color: blue,
+    description: `Contributing to reputable open-source projects is a great way to build experience in the tech industry without doing free work for clients.\n\n
+https://yourfirstpr.github.io/
+https://www.firsttimersonly.com/
+`
```

Comment:
Should probably use multiple lines for this.
Look at the other entries for the style guideline.

Reply:
In my case, I [used an array and joined](https://github.com/redxtech/devmod/blob/master/config/tags.js#L25-L42)

Reply:
Switch it to either of those (I'd probably go for the joined array) and I'll merge.

Reply:
Done.